pkg/ddl/ddl_api.go

```diff
@@ -3469,37 +3469,61 @@ func checkTwoRangeColumns(ctx sessionctx.Context, curr, prev *model.PartitionDef
 		// PARTITION p1 VALUES LESS THAN (10,20,'mmm')
 		// PARTITION p2 VALUES LESS THAN (15,30,'sss')
```

Comment:
Should we update this, so it is more clear that all columns are not increasing, but `5,10,'ggg'` is still less than `10,5,'mmm'` which is less than `10,30,'aaa'`?